src/api/InfluxDB.cpp

```diff
@@ -216,14 +229,18 @@ void vz::api::InfluxDB::send()
 		}
 		print(log_finest, "Reading buffer: timestamp %lld value %f", channel()->name(), it->time_ms(), it->value());
 		request_body.append(_measurement_name);
-		request_body.append(",uuid=");
-		request_body.append(channel()->uuid());
+		if (_send_uuid) {
+			request_body.append(",uuid=");
+			request_body.append(channel()->uuid());
+		}
 		if (!_tags.empty()) {
 			request_body.append(",");
 			request_body.append(_tags);
 		}
 		request_body.append(" value=");
-		request_body.append(std::to_string(it->value()));
+		std::stringstream value_str;
```

Comment:
if you're using a string stream already it might be easier to add the other data (value=....) as well and do once a request_body.append(stringstream.str())